server/src/main/java/org/elasticsearch/action/support/master/TransportMasterNodeAction.java

```diff
@@ -225,11 +275,24 @@ public void onTimeout(TimeValue timeout) {
         }
     }
 
+    private static final long DELAY_MILLIS = 100L;
```

Comment:
The parameters around this were chosen completely arbitrarily to be honest and mainly such that it stays at about the same order of magnitude as other retries we use in CCR.
It feels like we may want less aggressive retrying her though to take load off of master, but on the other hand master circuit breaking should be a temporary thing really and if it happens for a prolonged period of time it probably implies that the cluster is unstable in general and we get nothing from adding more latency here.